src/test/fuzz/crypto_diff_fuzz_chacha20.cpp

```diff
@@ -15,17 +27,17 @@ typedef unsigned char u8;
 
 #define ROTL32(v, n) (U32V((v) << (n)) | ((v) >> (32 - (n))))
 
-#define U8TO32_LITTLE(p)                                                       \
-  (((u32)((p)[0])) | ((u32)((p)[1]) << 8) | ((u32)((p)[2]) << 16) |            \
-   ((u32)((p)[3]) << 24))
+#define U8TO32_LITTLE(p)                                              \
```

Comment:
Can you break up this commit into two?:
- One commit making only formatting changes to DJB's code
- Another one adding the fuzz test changes.